### ISSUE TITLE
Determine the optimal hyperparameter values 

### ISSUE THREAD

**Issue by aynesss** [sayakpaul/SimCLR-in-TensorFlow-2#14]
Please can you tell me your opinion : it is enough to find a decreasing loss curve, to make sure that all parameters are optimal and my implementation is correct ?
